kilkari/src/main/java/org/motechproject/nms/kilkari/service/impl/CsrServiceImpl.java

```diff
@@ -281,6 +281,7 @@ public void processCallSummaryRecord(MotechEvent event) {
 
         CallRetry callRetry = callRetryDataService.findBySubscriptionId(subscriptionId);
         Subscription subscription = subscriptionDataService.findBySubscriptionId(subscriptionId);
+        resetWelcomeFlagInSubscription(subscription);
```

Comment:
Discussed with @llavoie  just now.

We will reset it in case of success call and the calls for which we have tried maximum number of retries.

Is that sound ok ?